server/src/main/java/org/eclipse/lsp/cobol/service/copybooks/PredefinedCopybooks.java

```diff
@@ -13,23 +13,24 @@
  *
  */
 
-package org.eclipse.lsp.cobol.service;
+package org.eclipse.lsp.cobol.service.copybooks;
 
 import com.google.common.collect.ImmutableMap;
 import lombok.experimental.UtilityClass;
+import org.eclipse.lsp.cobol.service.SQLBackend;
 
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import static org.eclipse.lsp.cobol.service.PredefinedCopybooks.Copybook.*;
+import static org.eclipse.lsp.cobol.service.copybooks.PredefinedCopybooks.Copybook.*;
 
 /** This util class encapsulates the logic of resolving the predefined variable names. */
 @UtilityClass
 public class PredefinedCopybooks {
   private static final Map<String, Copybook> PREDEFINED_COPYBOOKS =
-      ImmutableMap.of("SQLCA", SQLCA, "SQLDA", SQLDA, "DFHEIBLC", DFHEIBLC);
+      ImmutableMap.of("SQLCA", SQLCA, "SQLDA", SQLDA, "DFHEIBLC", DFHEIBLC, "PLABEL", PLABEL);
```

Comment:
As you see, it's a generated copybook, not predefined. Maybe it will be better to extend `CopybookAnalysisFactory.AnalysisTypes` with `GENERATED` item?